src/app/api/_api-services/offchain_db_service/firestore_service/index.ts

```
@@ -1625,4 +1629,168 @@ export class FirestoreService extends FirestoreUtils {
+			endsAt: pollData?.endsAt?.toDate?.(),
+			index: pollData?.index,
+			network: pollData?.network,
+			proposalType: pollData?.proposalType,
+			id: pollData?.id,
+			votes: [] // initially empty, will be fetched from the votes collection client side
+		};
+		return payload;
+	}
+
+	static async DeletePollVote({ network, proposalType, index, userId, pollId }: { network: ENetwork; proposalType: EProposalType; index: string; userId: number; pollId: string }) {
+		const pollSnapshot = await this.pollsCollectionRef()
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)